drivers/gles3/rasterizer_canvas_gles3.h

```diff
@@ -28,136 +28,50 @@
 /* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                */
 /*************************************************************************/
 
-#ifndef RASTERIZERCANVASGLES3_H
-#define RASTERIZERCANVASGLES3_H
+#pragma once
```

Comment:
@akien-mga are we starting to use #pragma once now?

Reply:
Ah that's probably me from initial implementation, I'll remove the #pragma once. :+1: